.travis.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
hmm, I think you're right @dianashk. That file was autogenerated, but it looks ok to remove.